pom.xml

```diff
@@ -82,9 +82,19 @@
                 <version>${okta.commons.version}</version>
             </dependency>
             <dependency>
-                <groupId>com.squareup.okhttp3</groupId>
-                <artifactId>okhttp</artifactId>
-                <version>${okhttp.version}</version>
+                <groupId>com.okta.commons</groupId>
+                <artifactId>okta-http-api</artifactId>
+                <version>${okta.commons.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.okta.commons</groupId>
+                <artifactId>okta-http-okhttp</artifactId>
```

Comment:
Verify that were this is used (likely the `impl` module) that there is a <scop>runtime</scope> added.
(technically, we shouldn't have a dependency on the implementation at all, but the previous version linked to ok-http directly, so keeping it makes upgrading to the next version easier.)

I have been thinking about way to get around the need to specify multiple dependencies, but that is for another issue

Reply:
Actually, it looks like the tests are failing, so maybe this dependency is missing?

Reply:
Fixed.